website/src/css/custom.css

```diff
@@ -4,27 +4,65 @@
  * work well for content-centric websites.
  */
 
+/* imported fonts */
+@font-face {
+  font-family: ShareTechMono;
+  src: url("../../static/fonts/Share_Tech_Mono/ShareTechMono-Regular.ttf");
```

Comment:
Let's use woff2 format for both fonts. This one is supported for the most of the browsers now and no need to use old font format.